### DISCUSSION TITLE
Max E-Term Heater directly to RS-485 on Nixie?

### DISCUSSION THREAD

**Discussion by fteston** [tagyoureit/nodejs-poolController#563] [RS485]
Max E-Term Heater directly to RS-485?
![image](https://user-images.githubusercontent.com/5831307/182669347-a84bd856-d0be-4840-bb73-775189f45059.png)

I have connected my Pool Pump, Intelligent VSF and Chlorinator IC-40 directly to my RS-485 adapter and heater.

![image](https://user-images.githubusercontent.com/5831307/182669563-70ea0c00-a037-4258-ab10-acdaed657409.png)

I am not sure why the heater is not working?
![image](https://user-images.githubusercontent.com/5831307/182669876-e0a668fa-0018-40ed-a13e-08c1b0e1d0a9.png)

![image](https://user-images.githubusercontent.com/5831307/182670000-153963bd-2d79-4b91-b87f-fde228ac6516.png)

**Comment by rstrouse** [comment]
Try changing the heater type to MasterTemp.  I think the protocols for these are the same.

**Reply by tagyoureit** [reply]
You also need to provide a pool temp to njsPC so it knows when to turn on/off the heater.

**Reply by fteston** [reply]
Hi , there anyway to setup a virtual pool temp to njsPC for a test?

**Reply by rstrouse** [reply]
If you install REM on a pc or Mac you can use a virtual ezo-rtd in the I2c tab. Then feed this temp to waterSensor1.
